src/test/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceServiceSpec.scala

```diff
@@ -151,6 +152,101 @@ class WorkspaceServiceSpec extends FreeSpec with ScalaFutures with ScalatestRout
       }
     }
 
+    "when calling POST on the workspaces/*/*/importEntities path" - {
+
+      "should 400 Bad Request if the first column header of the TSV is an unknown entity type" in {
+        val path = FireCloudConfig.Workspace.importEntitiesPathFromWorkspace(
+          MockWorkspaceServer.mockValidWorkspace.namespace.get,
+          MockWorkspaceServer.mockValidWorkspace.name.get)
```

Comment:
Thank you for using the configuration to get at the paths!!! One suggestion - since there are so many, it might be cleaner to make a val and use that in the individual tests:
```
val validWorkspacePath = FireCloudConfig.Workspace.importEntitiesPathFromWorkspace(
          MockWorkspaceServer.mockValidWorkspace.namespace.get,
          MockWorkspaceServer.mockValidWorkspace.name.get)
```